gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -107,7 +122,14 @@ public void start() throws IOException, InterruptedException {
     File gcdFolder = gcdPath.toFile();
     gcdFolder.deleteOnExit();
```

Comment:
Trying to understtand the logic from the code. Would love a short comment describing the logic....:
Do we unzip it at every invocation and delete it on exit?
Do we create 1 per project id?

Would it be possible to avoid the download/unzip and specific a local stable installation in the future?